tasks/2/index.js

```diff
@@ -2,7 +2,15 @@
 // be min, max and arithmetic mean of all objects in the array
 
 const getMinMaxAvg = (array) => {
-  // your code
+  const averageAllObj = array.reduce((acc, val) => (acc + val.age), 0) / array.length;
```

Comment:
you don't need `(`, `)` after `=>`.
You can write it so `(acc, val) => acc + val.age`